scripts/prepare_lambda_deployment.py

```diff
@@ -70,6 +77,56 @@ def patch_cloudwatch_resource(resource_type_name, property_name):
     return input_json
 
 
+def patch_resource_names(input_json: JSON) -> JSON:
+    # Some Chalice-generated resources have named ending in `-event`. The
+    # dash prevents generation of a fully qualified resource name (i.e.,
+    # with `config.qualified_resource_name`. Returns dict of old
+    # Terraform names and new Terraform names.
+    rename = {
+        'aws_cloudwatch_event_rule.indexercachehealth-event': 'indexercachehealth',
+        'aws_cloudwatch_event_target.indexercachehealth-event': 'indexercachehealth',
+        'aws_lambda_permission.indexercachehealth-event': 'indexercachehealth',
+        'aws_cloudwatch_event_rule.servicecachehealth-event': 'servicecachehealth',
+        'aws_cloudwatch_event_target.servicecachehealth-event': 'servicecachehealth',
+        'aws_lambda_permission.servicecachehealth-event': 'servicecachehealth',
+    }
+    replace = {k: '.'.join([k.split('.')[0], v]) for k, v in rename.items()}
+    literal = typing.TypeVar('literal', str, int)
```

Comment:
```suggestion
    T = typing.TypeVar('T', str, int)
```

https://www.python.org/dev/peps/pep-0008/#type-variable-names

If it needs to be hoisted to module scope, so be it.

Independently, "literal" is a bit misleading. "Literal" is a type of AST node, not a type of value. `int` refers to a type, `0` is an integer literal. `1+1` is an expression (another type of AST node) that evaluates to a value of type `int`. "Primitive" would be more accurate but with `T` we avoid the question.